gen/builders/state_zero.go

```diff
@@ -139,6 +140,18 @@ func (b *Builder) initializeZeroState(selector schema.Selector) {
 		})
 	}
 
+	rootVerifierID, err := address.NewFromString("t080")
```

Comment:
Errr, is this an arbitrary address? Or is this address also used in Lotus?

Reply:
This is what it's set to in Lotus master...

https://github.com/filecoin-project/lotus/blob/4311c96a4417bc2027402d2bfbfc8d2ecf4995ca/chain/gen/genesis/t06_vreg.go#L21

Reply:
...granted the variable name is `idk` 😂